sections/sprint-4/camera-handling.tex

```diff
@@ -0,0 +1,30 @@
+\section{Camera handling}
+This section will explain how the camera in the game is handled.
+Since the playing field can have a varying size the camera needs to be able to fit the whole playing field on the screen so that it can be rendered.
```

Comment:
```suggestion
Since the playing field can have a varying size the camera needs to be able to fit the whole playing field on the screen such that it can be rendered.
```